python/py-markdown-it-py/Portfile

```diff
@@ -2,34 +2,39 @@
 
 PortSystem          1.0
 PortGroup           python 1.0
-PortGroup           github 1.0
-
-github.setup        executablebooks markdown-it-py 2.0.0 v
```

Comment:
since you now removed the `github.setup` line you do need to set the `homepage` manually again

Reply:
Thanks done.